### ISSUE TITLE
@EliteLoser Any plans to push the 1.3.1 update to the PowerShell Gallery?

### ISSUE THREAD

**Issue by SQLGB** [EliteLoser/GetSTFolderSize#9]
@EliteLoser Any plans to push the 1.3.1 update to the PowerShell Gallery?

**Comment by EliteLoser** [comment]
Here we go. Terribly sorry about the delay in publishing, should have done it the same day, in theory.

```
PS C:\temp> cd foo
PS C:\temp\foo> Save-Module -Name GetSTFolderSize -Path .
PS C:\temp\foo> ipmo .\GetSTFolderSize\
PS C:\temp\foo> .\GetSTFolderSize\1.3.1\GetSTFolderSize.Tests.ps1
Import-Module : The specified module 'Pester' was not loaded because no valid module file was found in any module directory.
At C:\temp\foo\GetSTFolderSize\1.3.1\GetSTFolderSize.Tests.ps1:7 char:1
+ Import-Module -Name Pester -ErrorAction Stop
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : ResourceUnavailable: (Pester:String) [Import-Module], FileNotFoundException
+ FullyQualifiedErrorId : Modules_ModuleNotFound,Microsoft.PowerShell.Commands.ImportModuleCommand

PS C:\temp\foo> install-module -name pester -force -SkipPublisherCheck
PS C:\temp\foo> .\GetSTFolderSize\1.3.1\GetSTFolderSize.Tests.ps1

Describing Get-STFolderSize
  [+] Calculates the size of a directory with 5x 1024-byte files correctly using -ComOnly. 211ms
  [+] Calculates the size of a directory with 5x 1024-byte files correctly using -RoboOnly. 104ms
  [+] Gives the same result with COM and RoboCopy 17ms
  [+] Calculates the size of a directory tree with 5x4x 1024-byte files correctly using -ComOnly. 69ms
  [+] Calculates the size of a directory tree with 5x4x 1024-byte files correctly using -RoboOnly. 96ms
  [+] Gives the same result with COM and RoboCopy with a directory tree structure 6ms
  [+] Ignores *.ignored files that should be 5 kB in size in the test directory tree. 97ms
  [+] Ignores a specified directory name (with files containing 5 kB worth of files). 88ms
PS C:\temp\foo>
```

**Comment by EliteLoser** [comment]
Published v1.3.1.
